packages/phishing-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.1.0]
+
+### Added
+
+- Port `PhishingDetector` from `eth-phishing-detector`; add TypeScript types ([#4137](https://github.com/MetaMask/core/pull/4137))
+- Update `PhishingDetector` to support IPFS CID blocking ([#4465](https://github.com/MetaMask/core/pull/4465))
```

Comment:
nit: to emphasize that this is a new feature and not a behavior change.

```suggestion
- Add support for IPFS CID blocking to `PhishingDetector` ([#4465](https://github.com/MetaMask/core/pull/4465))
```

Reply:
Makes sense!